include/sr_enums.h

```diff
@@ -72,4 +72,14 @@ typedef enum {
     SRTensorTypeUint16  = 8  // 16-bit unsigned integer tensor type
 } SRTensorType;
 
+/*!
+*   \brief  Enumeration for logging levels
+*/
+typedef enum {
+    LLInvalid = 0, // Invalid or uninitialized logging level
```

Comment:
SmartSim progresses with:

Quiet
Info
Debug
Developer

It might be nice to be consistent with SmartSim so users and developers don't have to remember two different lexicons.

Reply:
Agree 100% that it makes sense to stay consistent. I've adjusted the levels to match SmartSim